NOTICE

```diff
@@ -0,0 +1,8 @@
+NervesTimeRtcNxp is open-source software licensed under the Apache License,
```

Comment:
```suggestion
NervesTime.RTC.NXP is open-source software licensed under the Apache License,
```

Sigh. I don't know how to automatically handle acronyms, so this needs to be special cased in the script. You'll see the other exceptions.